### ISSUE TITLE
No releases since April 2017

### ISSUE THREAD

**Issue by scooper91** [angular-ui/ui-select#2177]
There seem to be quite a few bug fixes and features on Master, which have not been released.  Please can you release a new version.

As the `dist` folder hasn't been updated, we can't even pull straight from GitHub. :(

**Comment by ErikAGriffin** [comment]
Yes please do a release and publish to npm.  None of these changes are actually hitting the users of this library, @Jefiozie @northteam @DarkIsDude

**Comment by omalyutin** [comment]
@Jefiozie @northteam @DarkIsDude 
It would be probably good: if original maintainers do not have time to manage this library, it worth giving minimal rights to other developers, who are interested in it. (at least accept pull requests and publish updates to npm)

I can take this responsibility, at least sometimes I can do something. :-)

Or new maintainers can create an "official fork" and mark current library as deprecated. Here is the example: https://github.com/jdorn/json-editor
